openghg/store/_boundary_conditions.py

```diff
@@ -232,7 +229,7 @@ def read_file(
 
             # Mop up and add additional keys to metadata which weren't passed to the parser
             boundary_conditions_data = self.update_metadata(
-                boundary_conditions_data, additional_input_parameters, additional_metadata
+                boundary_condition_data, additional_input_parameters, additional_metadata
```

Comment:
Maybe `boundary_condition_data` should be `boundary_conditions_data` here and above. Or the result of `self.update_metadata` should be `boundary_condition_data`